extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/PrivilegesBuilder.scala

```diff
@@ -122,6 +122,13 @@ object PrivilegesBuilder {
           buildQuery(child, privilegeObjects, projectionList, conditionList, spark)
         }
     }
+
+    // Should execute this for all plan and skip PermanentViewMaker
```

Comment:
where do we skip PermanentViewMaker？

Reply:
Remove this unnecessary comment

Reply:
So it's not skipped?

Reply:
Yea, no need to skip since it din't have an expression and we won't call `buildQuery` for its child.